yarn-project/aztec/src/cli/chain_l2_config.ts

```diff
@@ -0,0 +1,60 @@
+export type NetworkNames = 'testnet-ignition';
+
+export type L2ChainConfig = {
+  l1ChainId: number;
+  ethereumSlotDuration: number;
+  aztecSlotDuration: number;
+  aztecEpochDuration: number;
+  aztecProofSubmissionWindow: number;
+  testAccounts: boolean;
+  p2pEnabled: boolean;
+  p2pBootstrapNodes: string[];
+  registryAddress: string;
+};
+
+export const testnetIgnitionL2ChainConfig: L2ChainConfig = {
+  l1ChainId: 11155111,
+  ethereumSlotDuration: 12,
+  aztecSlotDuration: 36,
+  aztecEpochDuration: 32,
+  aztecProofSubmissionWindow: 64,
+  testAccounts: true,
+  p2pEnabled: true,
+  p2pBootstrapNodes: [],
+  registryAddress: '0x12b3ebc176a1646b911391eab3760764f2e05fe3',
+};
+
+export async function getBootnodes(networkName: NetworkNames) {
+  const url = `http://static.aztec.network/${networkName}/bootnodes.json`;
```

Comment:
Is it possible to turn on `https://` for this URL?

Reply:
We don't have it on these buckets.